marketplace/bundles/plan-marshall/skills/tools-integration-ci/standards/github-impl.md

```
@@ -401,18 +401,20 @@ gh auth status  # Uses GH_TOKEN automatically
-The script is invoked via the executor:
+The handler is reached through the provider-agnostic `ci` entry point — there is no
+`github` notation of its own; `ci.py` resolves the configured provider and dispatches
+to this implementation:
 
 ```bash
-python3 .plan/execute-script.py plan-marshall:tools-integration-ci:github <command> [args]
+python3 .plan/execute-script.py plan-marshall:tools-integration-ci:ci <command> [args]
 ```
 
 **Commands**:
 | Command | Description |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)